flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/CheckpointCoordinator.java

```diff
@@ -181,7 +188,16 @@ public class CheckpointCoordinator {
     private JobStatusListener jobStatusListener;
 
     /** A handle to the current periodic trigger, to cancel it when necessary. */
-    private ScheduledFuture<?> currentPeriodicTrigger;
+    private Future<?> currentPeriodicTrigger;
```

Comment:
It seems that `currentPeriodicTrigger` is always a `ScheduledFuture`. To avoid back-and-forth code change, would it be better to keep the existing code unless there is a good reason to change it?